### PR TITLE
Move `isRtl` from props to Redux for `PersonalRecentProjects`

### PR DESCRIPTION
**Pull request by joshlory**
Continue `isRtl` Redux cleanup.

### CODE REVIEW

File: apps/src/templates/projects/ProjectCardGrid.jsx [hunk 1]
```diff
@@ -32,28 +40,21 @@ const ProjectCardGrid = React.createClass({
...
 
   onSelectApp(appType) {
     this.setState({showAll: false, showApp: appType});
-  },
+  }
 
   viewAllProjects() {
     this.setState({showAll: true, showApp: ''});
-  },
+  }
```
Comment (davidsbailey) [inline apps/src/templates/projects/ProjectCardGrid.jsx]: `onSelectApp` and `viewAllProjects` do not appear to be properly bound to `this` here. I believe the preferred syntax is:
```javascript
  onSelectApp = appType => {
    this.setState({showAll: false, showApp: appType});
  }
```
@islemaster has done a bunch of these conversions and may be able to offer a second opinion. Here is one very long explanation, or you can skip to the end: http://reactkungfu.com/2015/07/why-and-how-to-bind-methods-in-your-react-component-classes/

Comment (joshlory) [reply apps/src/templates/projects/ProjectCardGrid.jsx]: Right, good catch!  I will fix.

**Comment by caleybrock** [comment]
So does this require a parent of https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/projects/ProjectWidgetWithData.jsx to be connected? Is that currently always true?

**Comment by joshlory** [comment]
It does.  All three usages of `<ProjectWidgetWithData>` are connected: [TeacherHomepage.jsx](https://github.com/code-dot-org/code-dot-org/blob/8760258734a1223d42319343e36bae8e8c2107d0/apps/src/sites/studio/pages/home/_homepage.js#L78), [StudentHomepage.jsx](https://github.com/code-dot-org/code-dot-org/blob/8760258734a1223d42319343e36bae8e8c2107d0/apps/src/sites/studio/pages/home/_homepage.js#L78), & [StageExtras.js](https://github.com/code-dot-org/code-dot-org/blob/8760258734a1223d42319343e36bae8e8c2107d0/apps/src/sites/studio/pages/scripts/stage_extras.js#L15).

**Comment by joshlory** [comment]
@davidsbailey verified fixed (green-red-green) with a shiny new unit test!

**Comment by davidsbailey** [comment]
Thanks for adding these tests! LGTM

**Pull request merged**